helpers.go

```diff
@@ -713,6 +713,15 @@ func NewInlineKeyboardButtonData(text, data string) InlineKeyboardButton {
 	}
 }
 
+// NewInlineKeyboardButtonLoginURL creates an inline keyboard button with text
+// which goes to a Login URL.
+func NewInlineKeyboardButtonLoginURL(text string, loginUrl LoginUrl) InlineKeyboardButton {
```

Comment:
I thought it was an helper method,. Can I have an example then?

Reply:
Yes, that looks like a good helper method! The issue is that the already implemented LoginURL code is on the develop branch instead of master, and this PR is based on master.

Reply:
Hang on, we missing important BotUsername,RequestWriteAccess and ForwardText fields.
In our case we use LoginURL with login from other bot, for ex.:)

Reply:
You're right. I'll fix it